widgets/class-teds-ACF-repeater-list-render.php

```diff
@@ -0,0 +1,281 @@
+                'type' => \Elementor\Controls_Manager::COLOR,
+                'default' => '#000',
+                'selectors' => [
+                    '{{WRAPPER}} h2' => 'color: {{VALUE}}',
+                ],
+                'condition' => [
+                    'show_title_list' => 'yes',
+                ],
+            ]
+        );
+        $this->add_group_control(
+            \Elementor\Group_Control_Typography::get_type(),
+            [
+                'name' => 'content_title_typographie',
+                'label' => esc_html__( 'Typographie du titre', 'textdomain' ),
+                'selector' => '{{WRAPPER}} h2',
+                'condition' => [
+                    'show_title_list' => 'yes',
+                ],
+            ]
+        );
+        $this->add_control(
+            'list_color',
+            [
+                'label' => esc_html__( 'Couleur des éléments de la liste', 'textdomain' ),
+                'type' => \Elementor\Controls_Manager::COLOR,
+                'default' => '#000',
+                'selectors' => [
+                    '{{WRAPPER}} li' => 'color: {{VALUE}}',
+                ],
+            ]
+        );
+        $this->add_group_control(
+            \Elementor\Group_Control_Typography::get_type(),
+            [
+                'name' => 'content_list_typographie',
+                'label' => esc_html__( 'Typographie des éléments de la liste', 'textdomain' ),
+                'selector' => '{{WRAPPER}} li',
+            ]
+        );
+        $this->end_controls_section();
+
+
+    }
+
+    /**
+     * Render the widget output on the frontend.
+     *
+     * Written in PHP and used to generate the final HTML.
+     *
+     * @since 1.0.0
+     *
+     * @access protected
+     */
+    protected function render()
+    {
+        global $wpdb;
+        global $post;
+        $settings = $this->get_settings_for_display();
+        if (!$settings['field_name']) {
+            return;
+        }
+
+        // Va chercher la liste des "ingrédients" (champs ACF défini sois-même) avec "rien" au début et "quelque chose" à la suite
+        $results = $wpdb->get_results("SELECT `meta_key`, `meta_value` FROM $wpdb->postmeta WHERE meta_key LIKE '" . $settings['field_name'] . "%' AND meta_key != '" . $settings['field_name'] . "' AND post_id = " . $post->ID . " ORDER BY meta_id ASC");
+        if (!count($results)) {
+            return;
+        }
+        // Affiche le titre uniquement s'il y a du contenu dans "ingrédients"
+        if ('yes' === $settings['show_title_list']) {
+            echo '<h2>' . $settings['title_list'] . '</h2>';
+        }
+        // Affiche le resultat pour chaque élément
+        foreach ($results as $result) {
+              echo '<ul style="padding-left: 0">
+                        <li style="list-style: none"><span>'. $settings['before_list'] .'</span>'. $result->meta_value .'</li>
+                    </ul>';
```

Comment:
You should add a class instead of use an inline style.
And add style in a css file (you can see others widgets to learn how to add a css for this widget)